app/views/layouts/application.html.erb

```diff
@@ -12,4 +12,16 @@
   <body>
     <%= yield %>
   </body>
+
+  <footer>
+    <div id="api_data">
+      <p>Contributors</p>
+        <ul>
+          <% @user_names.each do |name| %>
+            <li><%= name %></li>
+          <% end %>
+        </ul>
+        <p> Our total pull requests: <%= @pr_total %></p>
```

Comment:
We should probably consider having it all on one line instead of paragraphs so the footer doesn't take up too much room